aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -3493,33 +3493,11 @@ TORCH_IMPL_FUNC(_linalg_svd_out)(const Tensor& A,
   // the copy as a column major matrix to the backends.
   const auto info = at::zeros(IntArrayRef(A.sizes().begin(), A.sizes().end() - 2), A.options().dtype(kInt));
 
-  // Prepare S
-  const auto S_ = S.expect_contiguous();
-
-  // Prepare U / Vh
-  // U_ and Vh_ are just going to be accessed whenever compute_uv == true
-  const auto U_ready = !compute_uv || U.mT().is_contiguous();
-  const auto U_ = borrow_else_clone(U_ready, U, U, /*C-contig*/false);
```

Comment:
Nit. If you go on and move `ldl_factor` to use this, we would have all the operations that correctly handled strides using this pattern and we could remove `borrow_else_clone`. Then, I could port the new ones (https://github.com/pytorch/pytorch/pull/77634) to use this pattern.